src/table/utils/__tests__/selections-utils.spec.ts

```diff
@@ -200,27 +201,42 @@ describe('selections-utils', () => {
 
       afterEach(() => jest.clearAllMocks());
 
+      it('should return state unchanged, when type is selectMultiStart and mouseupOutsideCallback is undefined', () => {
+        const action = { type: SelectionActions.SELECT_MOUSE_DOWN, payload: { cell } } as SelectMouseDownAction;
+        const newState = reducer(state, action);
+        expect(newState).toEqual(state);
```

Comment:
Are you expected the same object reference? Such that `newState === state`? Then you should be using `.toBe()` method instead.

Reply:
good question, I don't remember how `useReducer` does that. I will try with toBe, that will give us the answer :)